server/src/main/java/io/openbytes/odi/interfaces/controller/api/DatasetController.java

```diff
@@ -31,4 +35,15 @@
         return Result.ok(service.getById(id));
     }
 
+
+    @ApiOperation(value = "Get dataset resources", notes = "get dataset resources.")
+    @GetMapping("/{dataset_name}/resources")
```

Comment:
rename "/{dataset_name}/resources"  to "/{dataset_name}/files" ?

Reply:
LGTM